examples/standards/norway/vehicles/submodel-VehicleType.xml

```diff
@@ -4,7 +4,7 @@
 	<Description>V</Description>
 	<SelfPropelled>true</SelfPropelled>
 	<TypeOfFuel>diesel</TypeOfFuel>
-	<EuroClass>Euro 6</EuroClass>
+	<!-- <EuroClass>Euro 6</EuroClass> -->
```

Comment:
Should we comment out code in examples?

Reply:
sometimes, but not here when it is without an explanation. I removed it.